CefSharp/RegisterCdmCallback.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright © 2010-2016 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using System;
+
+namespace CefSharp
+{
+    /// <summary>
+    /// Provides a simple callback implementation of <see cref="IRegisterCdmCallback"/> for use with Widevine CDM registration.
+    /// </summary>
+    public class RegisterCdmCallback: IRegisterCdmCallback
```

Comment:
Having the default implementation implement a `TaskCompletionSource` wrapper would be consistent with the rest of the codebase and allow you to add `Cef.RegisterWidevineCdmAsync`